tests/test_make_dirs.py

```diff
@@ -223,7 +223,7 @@ def test_make_sub_dir_no_tree(self, project):
         """
         Make sub directory only, check it is made and no lower level dirs exist
         """
-        project.make_sub_dir("ephys", "001", make_ses_tree=False)
+        project.make_sub_dir("ephys", "001", dont_make_ses_tree=True)
```

Comment:
I suggest renaming to "skip_ses_tree"

Reply:
good shout, now this option has been removed from later versions